wordle-web/src/components/ValidWords.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-expansion-panels v-model="drawer" color="black">
```

Comment:
Why use expansion panels here? You don't seem to have the functionality of the expansion panel.